packages/netsuite-adapter/src/transformer.ts

```diff
@@ -68,6 +68,8 @@ const getServiceIdFieldName = (type: ObjectType): string => {
   return isStandardType(type) ? SCRIPT_ID : PATH
 }
 
+const addBundlePrefix = (desiredName: string, type: ObjectType): string => (isBundleType(type) ? BUNDLE.concat(`_${desiredName}`) : desiredName)
```

Comment:
nit - use string formatting
```suggestion
const addBundlePrefix = (desiredName: string, type: ObjectType): string => (isBundleType(type) ? `${BUNDLE}_${desiredName}` : desiredName)
```